docs/output-contract.md

```
@@ -1,47 +1,37 @@
-- Deprecation follows the alias-only rule: old identifiers map to new ones; nothing is deleted.
-
-## Reference
-
-- Artifact layout: `contracts/docs/artifact-layout.md`
-- Finding payloads: `contracts/docs/finding-payload.md`
-- Identity and codes: `contracts/docs/identity-and-codes.md`
-- Cockpit comment ABI: `contracts/docs/cockpit-comment-abi.md`
-- Capabilities and missingness: `contracts/docs/capabilities-and-missingness.md`
+# depguard Output Contract
+
+## Problem
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)